core/controllers/ingest.py

```diff
@@ -52,46 +32,33 @@ def ingest(body, excel_file):
     :return: A tuple containing a string message and an integer HTTP status code.
     :raises ValidationError: If the data fails validation against the specified schema.
     """
-    source_type = body.get("source_type")  # required
-    place_names = body.get("place_names")
+    reporting_round = body.get(
```

Comment:
I think the docstring now needs updating for this (return looks wrong for example. Could you also add type hints while you're there?

Reply:
I think you forgot the type hints?